### ISSUE TITLE
Mono Wasm Compiler is unavaliable.

### ISSUE THREAD

**Issue by RaymondTracer** [0x414c49/mono-wasm-example#7]
Can you put up a mirror or link to somewhere else that hosts the compiler?

**Comment by divinci** [comment]
Did you find a link @RaymondTracer ?

**Comment by RaymondTracer** [comment]
> 
> 
> Did you find a link @RaymondTracer ?

 No, I actually figured out you can make standalone wasm apps by creating a "Blazor WebAssembly App" project in Visual Studio.

**Comment by divinci** [comment]
> > Did you find a link @RaymondTracer ?
> 
> No, I actually figured out you can make standalone wasm apps by creating a "Blazor WebAssembly App" project in Visual Studio.

Hi @RaymondTracer yes I wanted to make something without all the Blazor framework.  I suppose I will just try just directly modifying the Main entry point and see if it works.

**Comment by RaymondTracer** [comment]
> 
> 
> > > Did you find a link @RaymondTracer ?
> > 
> > 
> > No, I actually figured out you can make standalone wasm apps by creating a "Blazor WebAssembly App" project in Visual Studio.
> 
> Hi @RaymondTracer yes I wanted to make something without all the Blazor framework. I suppose I will just try just directly modifying the Main entry point and see if it works.


@divinci Oh, I see, I'm not sure if I could help with that, my end goal was getting a standalone wasm app in C# (Blazor or not) but I figured I'd keep this issue open until the repo owner decides to do something about it (either close the repo and tell people to use Blazor, or fix the download link).

**Comment by 0x414c49** [comment]
The repo owner is here. I cannot find a new download link. I'll try to find another way of doing it.

**Comment by divinci** [comment]
> The repo owner is here. I cannot find a new download link. I'll try to find another way of doing it.

Thanks Ali - any joy?

**Comment by parthipanr** [comment]
any one found the new download link?
